src/app/components/shared/model-cards/card/card.component.scss

```diff
@@ -16,7 +16,7 @@
     border-bottom: var(--card-border);
 
     a {
-      display: inline-block;
+      display: block;
```

Comment:
so this is the magic that stops the card from collapsing?

maybe add a comment that this is load bearing

Reply:
nit: s/disabled/display/